src/platform/packages/private/kbn-esql-editor/src/helpers.ts

```diff
@@ -310,3 +312,35 @@ export const onKeyDownResizeHandler = (
     }
   }
 };
+
+export const getEditorOverwrites = (theme: EuiThemeComputed) => {
+  return css`
+    .monaco-hover {
+      display: block !important;
+    }
+    .margin-view-overlays .line-numbers {
+      color: ${theme.colors.textDisabled};
+    }
+    .current-line ~ .line-numbers {
+      color: ${theme.colors.textSubdued};
+    }
+
+    .suggest-widget,
+    .suggest-details-container {
+      @include euiBottomShadow;
```

Comment:
This seems to be SCSS code. The replacement for `@include euiBottomShadow` is `useEuiShadow('l')`. See https://github.com/elastic/eui/discussions/6828#discussioncomment-10825360.

```suggestion
      ${useEuiShadow('l')};
```

Since `useEuiShadow` is a React hook, you will need to convert the `getEditorOverwrites` function to a custom React hook. My understanding is that you can simply rename this helper function to have a `use` prefix to do that.

Reply:
Cool - I didn't notice the part of that doc about the non-hook utility. I'm glad you found a way to make it simpler!